jupyterhub/templates/hub/secret.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   name: hub-secret
 type: Opaque
 data:
+  # {{ required "proxy.secretToken must be a 32byte random string generated with openssl rand -hex 32" .Values.proxy.secretToken }}
```

Comment:
` ticks around "openssl rand -hex 32". Also should there be a space between 32 and byte? Sorry if I'm nitpicking here

Reply:
Done